### DISCUSSION TITLE
SSO Flow Question - Registering a user

### DISCUSSION THREAD

**Discussion by mark-james** [directus/directus#14862] [Archive]
Hi,
I've successfully setup SSO using the microsoft connector. I can register a new user via SSO. However if I create the user prior to their first login I get an error of 'Duplicate Value Detected'.

Is there a way of logging a user in via SSO that has been created manually?

Thanks
Mark

**Comment by rijkvanzanten** [comment]
Only if the user's auth provider is set to the correct provider, and the external identifier is set ahead of time. You can't currently login as a email+password user through sso for example, see https://github.com/directus/directus/issues/9331 for more on that 🙂
